azure/template.json

```diff
@@ -79,12 +79,431 @@
         },
         "sharedManagementResourceGroup": {
             "type": "string"
+        },
+        "deploySandbox": {
+            "type": "bool"
+        },
+        "activeSandboxAppServicePlanName": {
+            "type": "string"
+        },
+        "aseHostingEnvironmentName": {
+            "type": "string"
+        },
+        "aseResourceGroup": {
+            "type": "string"
+        },
+        "aspSize": {
+            "type": "string"
+        },
+        "aspInstances": {
+            "type": "string"
+        },
+        "externalApiActiveSandboxAppServiceName": {
+            "type": "string"
+        },
+        "certificateName": {
+            "type": "string"
+        },
+        "internalApiSandboxAppServiceName": {
+            "type": "string"
+        },
+        "internalApiSandboxCustomHostName": {
+            "type": "string"
+        },
+        "sandBoxFunctionAppNames": {
+            "type": "array"
+        },
+        "sharedSqlResourceGroup": {
+            "type": "string"
+        },
+        "secondaryDatabaseName": {
+            "type": "string"
+        },
+        "sharedSqlServerName": {
+            "type": "string"
+        },
+        "databaseSkuName": {
+            "type": "string"
+        },
+        "databaseTier": {
+            "type": "string"
+        },
+        "logAnalyticsSubscriptionId": {
+            "type": "string"
+        },
+        "logAnalyticsResourceGroup": {
+            "type": "string"
+        },
+        "logAnalyticsWorkspaceName": {
+            "type": "string"
+        },
+        "externalActiveApiSandboxCustomHostname": {
+            "type": "string"
         }
     },
     "variables": {
         "InstanceName": "[if(equals(toUpper(parameters('environmentName')),'PROD'),'',parameters('environmentName'))]"
     },
     "resources": [
+        {
+            "apiVersion": "2017-05-10",
+            "name": "sandbox-app-service-plan",
+            "type": "Microsoft.Resources/deployments",
+            "condition": "[equals(parameters('deploySandbox'), 'true' )]",
```

Comment:
"condition" takes a bool value, so this can just be set to [parameters('deploySandbox')] now